cypress.config.js

```diff
@@ -62,6 +62,7 @@ const integrationConfig = {
 const endToEndConfig = {
   baseUrl: 'https://nextjs-pokedex-lemon.vercel.app/',
   specPattern: 'cypress/e2e/**/*.cy.{js,jsx,ts,tsx}',
+  trashAssetsBeforeRuns: true,
```

Comment:
:bulb: Ensuring that artifacts are trashed inbetween the preview workflows runs of integration and e2e tests.